### PR TITLE
Adding new option(R) to the origin in Make Date Dialog

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes#5429

@africanmathsinitiative/developers  This is ready for review

### CODE REVIEW

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -103,7 +103,7 @@ Public Class dlgMakeDate
         ucrInputFormat.SetItems(dctDateFormat)
         ucrInputFormat.SetDropDownStyleAsEditable(bAdditionsAllowed:=True)
 
-        ucrInputOrigin.SetItems({"Excel", "Gregorian", "Julian Day Number", "Specify"})
+        ucrInputOrigin.SetItems({"R", "Excel", "Gregorian", "Julian Day Number", "Specify"})
```
Comment (shadrackkibet) [inline instat/dlgMakeDate.vb]: @dannyparsons  mentioned the change of item labels i.e **Excel (1899/12/30)** in the issue, others should follow the same format as well.

Comment (dannyparsons) [reply instat/dlgMakeDate.vb]: Yes all of the special formats in the list should have the date added.

Comment (shadrackkibet) [reply instat/dlgMakeDate.vb]: @Wycklife could you please make this small edits?

**Comment by maxwellfundi** [comment]
@Wycklife Please could you make this change before this is merged in?

**Pull request closed without merging**